test-support/helpers/ember-tooltips.js

```diff
@@ -222,3 +222,20 @@ export function assertTooltipSpacing(assert, options) {
         - Tooltip should be on the ${side} side of the target: ${isSideCorrect}.
         - On the ${side} side of the target, the tooltip should be ${spacing}px from the target but it was ${actualSpacing}px`);
 }
+
+export function assertTooltipContent(assert, options = {}) {
+  const { contentString } = options;
+
+  if (contentString === undefined) {
```

Comment:
If we use `isEmpty` then a user would not be able to test the case where `contentString` is an empty string.